runtime/doc/api.txt

```diff
@@ -3093,13 +3109,14 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
       • {config}  Map defining the window configuration. Keys:
                   • relative: Sets the window layout to "floating", placed at
                     (row,col) coordinates relative to:
+                    • "" Non-floating
```

Comment:
Any reason not to allow omitting `relative` (that was always what I assumed we'd do since we introduced `nvim_open_win`)?

Reply:
No, not particularly, I didn't think that through much. Should omitting `relative` resolve to a split window, the same as `relative=""`?

Reply:
Maybe `win_get_config` shouldn't return anything for `relative` in split wins then? Currently it returns an empty string.